File: Codingbat/Python/List-2/sum67.py
def sum67(nums):
  status = True 
  total = 0 
  
  for num in nums: 
    if num == 6: 
      status = False 
    
    if status == True: 
      total += num 
      
    if num == 7: 
      status = True 
      
  return total 

File: Codingbat/Java/Array-2/sum67.java
public int sum67(int[] nums) {
  int total = 0; 
  boolean status = true;
  
 for(int i = 0; i<nums.length; i++){
   if(nums[i] == 6){
     status = false;
   } 
   
   if (status == true){
     total += nums[i];
   }
   
   if(nums[i] == 7) {
     status = true;
   }
 }
 return total;
}

